android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -727,7 +729,15 @@ public void handleJsonEvent(final JsonReadEvent jsonReadEvent) {
 
                     OpenEventApp.postEventOnUIThread(new MicrolocationDownloadEvent(true));
                     break;
-                } default:
+                }case ConstantStrings.SESSION_TYPES: {
+                    List<SessionType> sessionTypes = objectMapper.readValue(json, new TypeReference<List<SessionType>>() {});
```

Comment:
@arp95 Please look at https://github.com/iamareebjamal/open-event-android-sdk/pull/19/files#diff-101323bdb4e416731a603a3a109aba5bL55

Previous implementation has bug on parsing List<T>. Thanks.

Change it at other places also.